apps/src/aichat/views/bot/Bot.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import moduleStyles from '../chatWorkspace.module.scss';
+import classNames from 'classnames';
+
+const botImagePath = `/blockly/media/aichat/ai-bot-default.svg`;
+const hand0ImagePath = `/blockly/media/aichat/ai-bot-hand-0.png`;
+const hand1ImagePath = `/blockly/media/aichat/ai-bot-hand-1.png`;
+const foot0ImagePath = `/blockly/media/aichat/ai-bot-foot-0.png`;
+const foot1ImagePath = `/blockly/media/aichat/ai-bot-foot-1.png`;
+
+export interface BotState {
+  head: any;
```

Comment:
The type is `head: [number, number]` etc.  It could be extracted to a named type, ie.
```
type Point = [number, number];

export interface BotState {
  head: Point;
  body: Point;
...
```